lib/envy.ex

```diff
@@ -85,7 +85,7 @@ defmodule Envy do
     end)
   end
 
-  defp blank_entry?(string) do
-    string == ""
+  defp invalid_entry?(string) do
+    string == "" || String.match?(string, ~r(^\s*#))
```

Comment:
Agreed, I'll split these up into separate functions and chain them using reject.